packages/amplify-e2e-tests/src/__tests__/diagnose.test.ts

```diff
@@ -10,10 +10,10 @@ import {
   diagnoseSendReport,
   diagnoseSendReport_ZipFailed,
 } from '@aws-amplify/amplify-e2e-core';
+import { extract } from '@aws-amplify/amplify-cli-core';
```

Comment:
Do we need to use this in the test package? I'd prefer to put the extract function in `amplify-provider-cloudformation` so that it doesn't become a public member of cli-core. But if we do need to share it across packages, then I think the structure you've got makes sense.

Reply:
I was trying to get rid of this from yarn_lock to make sure nobody else is using it either directly or transitively.